plenum/test/view_change/test_view_change_no_quorum.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from plenum.test.view_change.helper import ensure_view_change
+from plenum.test.helper import stopNodes
+
+
+def test_view_change_no_quorum(nodeSet, up, looper):
+    """
```

Comment:
I understand this test is used to ensure quorum of `n-f`, can you merge this with `test_view_change_n_minus_f_quorum` because `test_view_change_n_minus_f_quorum` is not a complete test